interactions/api/http/guild.py

```diff
@@ -280,6 +280,15 @@ async def create_guild_from_guild_template(
             Route("POST", f"/guilds/templates/{template_code}", json=payload)
         )
 
+    async def get_guild_template(self, template_code: str) -> dict:
+        """
```

Comment:
```suggestion
        """
        .. versionadded:: 4.4.0
        
```

Reply:
What is this

Reply:
something for docs
![grafik](https://user-images.githubusercontent.com/88881326/201429221-870c8a0e-a486-4d14-aff3-15abee1523ad.png)